### PR TITLE
155533490 ulkoiset rajapinnat  osio muutoksia 26.2. läpikäynnin pohjalta

### PR DESCRIPTION
**Pull request by K1ll3rF0x**
# Fixed
* Removed global <p> element styling completely. Fixed some styling issues related to rendered markdown.
* Added some translation fixes for #155062911.
* Fix a bug in chip-input auto-select feature. Now supporting autocomplete vector-map datasources in addition to plain strings.

# Added
* Change interfaces data-content field to chip input, format field to autocomplete and remove AGPL suggestion from licesenses field.

### CODE REVIEW

File: ote/src/cljs/ote/ui/form_fields.cljs [hunk 1]
```diff
@@ -332,6 +332,11 @@
                    (when on-blur (on-blur event)))
         :on-request-add handle-add!
         :on-request-delete handle-del!}
+       ;; Define suggestions data element format.
```
Comment (tatut) [inline ote/src/cljs/ote/ui/form_fields.cljs]: I don't understand these comment lines. Are they meant for users of the `:chip-input` form field type or just a reminder about the internals?

File: ote/src/cljs/ote/ui/mui_chip_input.cljs [hunk 2]
```diff
@@ -47,8 +47,15 @@
                    (if @auto-select?
```
Comment (tatut) [inline ote/src/cljs/ote/ui/mui_chip_input.cljs]: Can we make a drive-by fix and remove the auto-select atom? I don't see the need for it. It is just reset! on render. If it is needed, for some reason, document it.

File: ote/resources/public/css/styles.css [hunk 3]
```diff
@@ -32,10 +32,12 @@ h2 {
...
 }
 
-p {
-    line-height: 21px;
-    -webkit-margin-before: 0;
-    -webkit-margin-after: 0;
+
+/* Rendered translations are wrapped with <p> by Marked JS library.
+   Remove the margins of the first P-element */
+.translation-markdown > p{
```
Comment (rckopteri) [inline ote/resources/public/css/styles.css]: could add empty space before {

File: ote/src/cljs/ote/ui/form_fields.cljs (hunk 1 above)
Comment (K1ll3rF0x) [reply ote/src/cljs/ote/ui/form_fields.cljs]: Reminder about the internal Autocomplete component behaviour.
